test/models/track_test.rb

```diff
@@ -25,6 +25,45 @@ class TrackTest < ActiveSupport::TestCase
     assert_equal 1, AudioFile.count
   end
 
+  test 'should be able to merge two tracks with one resulting audio_file' do
+    a1 = create(:track)
+    a2 = create(:track, :with_audio_file)
+
+    a1.merge(a2)
+
+    assert_equal 1, AudioFile.count
+    assert a1.reload.audio_file.present?
+  end
+
+  test 'should be able to merge two tracks with zero resulting audio_files' do
```

Comment:
I don't see how this is the behaviour we want. Why would we want to end up with a track with audio?

Reply:
I just added tests documenting the current behaviour of the method. If we want to change that behaviour (and I agree that the behaviour could be better) that should happen in a different PR.

Reply:
👍